### DISCUSSION TITLE
Trouble running on linux

### DISCUSSION THREAD

**Discussion by Eckstrom13** [sn4k3/UVtools#525] [General]
Hello im having trouble running either the loose program or the appimage each giving me the same error.

```
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet---crash-report does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
```
  
Im using Arch and have all the listed prerequisites installed. Any help would be appreciated.

**Comment by sn4k3** [comment]
What arch linux you run and the version?

**Comment by Eckstrom13** [comment]
Normal Arch and im on the latest kernel 5.18.14-arch1-1.

**Reply by sn4k3** [reply]
And which UI manager do you use?
I never tried UVtools on arch, but i does work on manjaro

**Comment by Eckstrom13** [comment]
Im using Qtile. But that shouldn't change much since im launching it through the terminal no?

**Reply by sn4k3** [reply]
Window manager matters. But i can't help since i'm not linux user and no experience with arch and windows managers. 
I know that on manjaro it works, try to install xfce or Gnome for example to try

**Comment by muesli** [comment]
Hey @Eckstrom13,

I've created a `uvtools-bin` package in the AUR earlier today. Could you try that one?

```bash
yay -S uvtools-bin
```
